systemtests/src/test/java/com/github/streamshub/systemtests/system/AbstractST.java

```diff
@@ -72,12 +81,28 @@ public abstract class AbstractST {
     }
 
     @BeforeAll
-    void beforeAll() {
-        LOGGER.info("=========== AbstractST - BeforeAll ===========");
+    void setupTestSuite() {
+        LOGGER.info("=========== AbstractST - BeforeAll - Setup TestSuite ===========");
```

Comment:
Maybe `TestVisualSeparatorExtension` can be used from test-frame? Depends if you want it that way. Just wanted to mention it, so you are aware of it.

Reply:
You can see that even when separator logs the method name, it is more visible when the line you marked is logged as well. Or at least that is my opinion.

Reply:
I guess it's up to you then :)

Reply:
Right, i'd keep this one in.